draft-ietf-ace-revoked-token-notification.md

```diff
@@ -851,6 +912,8 @@ Once registered at the AS, the administrator or registered device can send a GET
 
 In case the request is successfully processed, the AS replies with a response specifying the CoAP response code 2.05 (Content). In particular, if the AS supports diff queries but not the "Cursor" extension (see {{sec-trl-endpoint-supporting-diff-queries}} and {{sec-trl-endpoint-supporting-cursor}}), then the payload of the response is formatted as defined in {{ssec-trl-full-query}} or in {{ssec-trl-diff-query}}, in case the GET request has yielded the execution of a full query or of a diff query of the TRL, respectively. Instead, if the AS supports both diff queries and the related "Cursor" extension, then the payload of the response is formatted as defined in {{sec-using-cursor}}.
 
+In case a requester does not receive a response from the TRL endpoint or it receives an error response from the TRL endpoint, the requester does not make any assumption or draw any conclusion regarding the revocation or expiration of its pertaining access tokens.
```

Comment:
You could mention a retry policy here.

Reply:
Now addressed at: https://github.com/ace-wg/ace-revoked-token-notification/pull/13/commits/0db3c7be32172151dc139eb8255a2e880cffa432